food.js

```diff
@@ -0,0 +1,31 @@
+var DogFood = (function (originalDogFood) {
+  var privateDogFood;
+  // console.log(this.responseText);
+
+    originalDogFood.loadDogFood = function (){
+      var loadDog = new XMLHttpRequest();
+      var loadCat = new XMLHttpRequest();
+
+      loadDog.addEventListener("load", function () {
+        privateDogFood = JSON.parse(this.responseText).dog_brands;
+        console.log("privateDogFood", privateDogFood);
+
+        DogFood.setDogFood(privateDogFood);
+
+      });
+
+      loadDog.open("GET", "dog.json");
+      loadDog.send();
+      loadCat.open("GET", "cat.json");
+      loadCat.send();
```

Comment:
Making both these calls sequentially like this won't work as expected.  Since the ```load``` function is fired asynchronously, there's no guarantee that both of them won't arrive at the same time, causing the load function to just give up.

The way your code is currently written, with both cat and dog data using the same load function, you'd need to make the XHR call to one, and in that one's load function, make the call to the other.  Since both call the same load function, you'd need to check to make sure you only call the second XHR when the first one fires the load function, otherwise when the second XHR loads, it'll call itself again, and bam, infinite loop.